Buildings/Resources/Scripts/Dymola/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/Validation/PIDGain.mos

```diff
@@ -0,0 +1,4 @@
+simulateModel("Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.Validation.PIDGain", method="Cvode", tolerance=1e-06, resultFile="PIDGain");
+createPlot(id=1, position={15, 15, 1484, 920}, y={"PIDGain.kp", "PIDGain.T", "PIDGain.L"}, range={0.0, 1.0, 0.0, 2.5}, grid=true, subPlot=101, colors={{28,108,200}, {238,46,47}, {0,140,72}});
```

Comment:
should not be `y={"PIDGain.kp", "PIDGain.T"...`, but `y={"PIDGai.kp", "PIDGai.T"...`.

Reply:
Please run each of the validation model, by clicking `simulate and plot`. It will show you if the model can run and if the variables set in the .mos script are correct.